code/go/0chain.net/blobbercore/readmarker/entity.go

```diff
@@ -106,7 +106,6 @@ func GetLatestReadMarkerEntity(ctx context.Context, clientID string) (*ReadMarke
 }
 
 func SaveLatestReadMarker(ctx context.Context, rm *ReadMarker, isCreate bool) error {
-
 	var (
```

Comment:
If there are no meaningful changes to entity.go, revert changes and remove from PR.